### PR TITLE
1010CG Design Update

### PR DESCRIPTION
**Pull request by LinnJS**
## Description
- Update 1010CG according to final design iteration 
- Remove country from address
- Add confirm email
- Add modals for extra content

[Designs](https://vsateams.invisionapp.com/share/9XWSQZ6SMG5)

## Screenshots
image

## Acceptance criteria
- [ ] Layout and content matches designs

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/caregivers/components/AdditionalInfo/index.jsx [hunk 1]
```diff
@@ -1,16 +1,100 @@
...
 
+export const LearnMoreFamilyCaregiver = () => {
+  const [isModalVisible, toggleModal] = useState(false);
+
+  return (
+    <>
+      <Modal
+        title="Primary Family Caregiver"
+        id="primary-family-caregiver-info"
+        cssClass="caregiver-modal"
```
Comment (bradl3yC) [inline src/applications/caregivers/components/AdditionalInfo/index.jsx:19]: I really wish we could switch this to className or something similar - not related to your work but this is an odd prop name

Comment (LinnJS) [reply src/applications/caregivers/components/AdditionalInfo/index.jsx:19]: Yeah I totally agree, but should isolate the change in another PR, but I feel ya fo sho

**Review by bradl3yC — APPROVED** [review]
LGTM!

File: src/applications/caregivers/containers/IntroductionPage.jsx [hunk 2]
```diff
@@ -88,73 +108,76 @@ const IntroductionPage = ({ route, router }) => {
...
             </p>
 
-            <div className="vads-u-margin-bottom--2p5">
-              <PowerOfAttorneyInfo />
-            </div>
-            <RepresentativeInfo />
+            <p>
+              <b>Note:</b> If the Veteran isn’t enrolled in VA health care or is
+              currently on active duty with a medical discharge, they’ll need to
+              fill out an Application for Health Benefits (VA Form 10-10EZ).
```
Comment (mr0sari0) [inline src/applications/caregivers/containers/IntroductionPage.jsx:119]: In your design file, "Application for Health Benefits" is a link.

File: src/applications/caregivers/containers/IntroductionPage.jsx [hunk 3]
```diff
@@ -23,62 +19,86 @@ const IntroductionPage = ({ route, router }) => {
...
+                  or a Veterans Service Organization
+                </li>
+              </ul>
+
+              <a
+                href="https://www.va.gov/disability/get-help-filing-claim/"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                Get filing our your application
```
Comment (mr0sari0) [inline src/applications/caregivers/containers/IntroductionPage.jsx:100]: The design file says "Get help filling out your application".

File: src/applications/caregivers/containers/IntroductionPage.jsx (hunk 2 above)
Comment (LinnJS) [reply src/applications/caregivers/containers/IntroductionPage.jsx]: Just fixed with design good catch

File: src/applications/caregivers/containers/IntroductionPage.jsx (hunk 3 above)
Comment (LinnJS) [reply src/applications/caregivers/containers/IntroductionPage.jsx:100]: Just did a design adit with @jonathanjnelson just correct these, thanks for keeping a close eye!

Comment (LinnJS) [reply src/applications/caregivers/containers/IntroductionPage.jsx]: Actually we didn't get this one after all, yay team work!

File: src/applications/caregivers/components/NeedHelpFooter/index.jsx [hunk 4]
```diff
@@ -1,55 +1,42 @@
...
           rel="noopener noreferrer"
+          href="https://www.caregiver.va.gov/support/New_CSC_Page.asp"
         >
-          {links.caregiverHelpPage.link}
-        </a>
-      </section>
-
-      <section className={sectionClassNames}>
-        <span>
-          You can contact the National Caregiver Support Line by calling:
```
Comment (mr0sari0) [inline src/applications/caregivers/components/NeedHelpFooter/index.jsx:35]: Looks like there should be a space between "calling:" and the phone number. Something like `{' '}` should work.

**Comment by mr0sari0** [comment]
I didn't see these lines in your PR but for the Veteran/service member information page:

- add a period after "Please complete all of the following information"
- update "First name" to "Veteran's first name"
- update "Middle name" to "Veteran's middle name"
- update "Last name" to "Veteran's last name"
- update "Date of birth" to "Veteran's date of birth"
- update "Sex" to "Veteran's sex"

File: src/applications/caregivers/definitions/caregiverUI.js [hunk 5]
```diff
@@ -217,3 +254,60 @@ export default {
...
+      },
+    },
+  ],
+});
+
+export const addressWithoutCountry = {
+  'ui:title': ' ',
+  'ui:order': ['street', 'street2', 'city', 'state', 'postalCode'],
+  street: {
+    'ui:title': 'Street',
```
Comment (mr0sari0) [inline src/applications/caregivers/definitions/caregiverUI.js:289]: The design has this field's title as "Veteran's current street address".

File: src/applications/caregivers/definitions/caregiverUI.js [hunk 6]
```diff
@@ -40,41 +37,51 @@ export default {
       'ui:title': 'Email Address',
```
Comment (mr0sari0) [inline src/applications/caregivers/definitions/caregiverUI.js:37]: The design has this field's title as "Veteran's email address".

File: src/applications/caregivers/definitions/caregiverUI.js [hunk 7]
```diff
@@ -40,41 +37,51 @@ export default {
...
       'ui:widget': 'email',
```
Comment (mr0sari0) [inline src/applications/caregivers/definitions/caregiverUI.js:38]: The design has the email field as required.

File: src/applications/caregivers/definitions/caregiverUI.js [hunk 8]
```diff
@@ -40,41 +37,51 @@ export default {
...
     },
+    confirmationEmailUI: {
```
Comment (mr0sari0) [inline src/applications/caregivers/definitions/caregiverUI.js:40]: The design has the confirmation email field as required.

**Review by maharielrosario — COMMENTED** [review]
For the contact information page, the design has "Contact information" bolded underneath the chapter name.

**Comment by mr0sari0** [comment]
For the contact information page:

- update the field title "Primary Telephone Number (Including Area Code)" to "Veteran's primary telephone number (including area code) and make this field required"
- update the field title "Alternate Telephone Number (Including Area Code)" to "Veteran's alternate telephone number (including area code)"

File: src/applications/caregivers/definitions/caregiverUI.js (hunk 8 above)
Comment (LinnJS) [reply src/applications/caregivers/definitions/caregiverUI.js]: Was technically already required due to not being able to continue without the validation passing, went ahead and added it to the required array anyways.

File: src/applications/caregivers/components/NeedHelpFooter/index.jsx (hunk 4 above)
Comment (LinnJS) [reply src/applications/caregivers/components/NeedHelpFooter/index.jsx:35]: I think this is fixed? going to run through and remove these spaces and use CSS classes.

File: src/applications/caregivers/components/AdditionalInfo/index.jsx (hunk 1 above)
Comment (Mottie) [reply src/applications/caregivers/components/AdditionalInfo/index.jsx:28]: There is a lot of repeated content here. Would you find a way to make it more DRY?

File: src/applications/caregivers/components/AdditionalInfo/index.jsx [hunk 9]
```diff
@@ -1,27 +1,120 @@
...
+          </ul>
+        </>
+      </Modal>
+
+      <button
+        type="button"
+        className="va-button-link"
+        onClick={() => toggleModal(!isModalVisible)}
+      >
+        Learn more about what this means
```
Comment (Mottie) [inline src/applications/caregivers/components/AdditionalInfo/index.jsx:58]: I know this is in the design, but I don't think this text is good for accessibility. It's weird that it's a modal inside a dropdown. Has @jenstrickland seen this? ([design page](https://vsateams.invisionapp.com/share/9XWSQZ6SMG5#/screens/413997345)).

File: src/applications/caregivers/config/form.js [hunk 10]
```diff
@@ -296,37 +346,45 @@ const formConfig = {
       },
     },
     secondaryCaregiversChapter: {
-      title: 'SECONDARY CAREGIVERS',
+      title: 'Secondary Family Caregivers Information',
```
Comment (Mottie) [inline src/applications/caregivers/config/form.js:349]: Lower case `i` in `Information`?

File: src/applications/caregivers/config/form.js [hunk 11]
```diff
@@ -296,37 +346,45 @@ const formConfig = {
...
               },
             },
           },
         },
-        secondaryOneCaregiver: {
+        secondaryCaregiverOne: {
           path: 'secondaryOne-caregiver-1',
-          title: 'Secondary Caregiver Information',
-          depends: formData => hasSecondaryOneCaregiver(formData),
+          title: 'Secondary Family Caregiver One Information',
```
Comment (Mottie) [inline src/applications/caregivers/config/form.js:369]: I didn't see `One` in the design; but I did see `(2)`? Is that the index?

File: src/applications/caregivers/config/form.js [hunk 12]
```diff
@@ -296,37 +346,45 @@ const formConfig = {
...
+          depends: formData => hasSecondaryCaregiverOne(formData),
           uiSchema: {
-            'ui:description': SecondaryCaregiverInfo,
+            'ui:description': SecondaryCaregiverInfo({
+              pageTitle: 'Secondary Caregiver One Information',
```
Comment (Mottie) [inline src/applications/caregivers/config/form.js:373]: Here too

File: src/applications/caregivers/config/form.js [hunk 13]
```diff
@@ -296,37 +346,45 @@ const formConfig = {
...
+            [secondaryCaregiverFields.secondaryOne.ssn]: ssnUI(
+              'Secondary One Family Caregiver',
+            ),
+            [secondaryCaregiverFields.secondaryOne.dateOfBirth]: dateOfBirthUI(
+              'Secondary One Family Caregiver',
+            ),
+            [secondaryCaregiverFields.secondaryOne.gender]: genderUI(
+              'Secondary One Family Caregiver',
+            ),
           },
```
Comment (Mottie) [inline src/applications/caregivers/config/form.js:388]: Maybe I'm missing something?

File: src/applications/caregivers/config/form.js [hunk 14]
```diff
@@ -339,75 +397,103 @@ const formConfig = {
...
+              'Secondary Two Family Caregiver',
+            ),
+            [secondaryCaregiverFields.secondaryTwo.gender]: genderUI(
+              'Secondary Two Family Caregiver',
+            ),
+            [secondaryCaregiverFields.secondaryTwo
+              .address]: addressWithoutCountryUI(
+              'Secondary Two Family Caregiver',
+            ),
           },
```
Comment (Mottie) [inline src/applications/caregivers/config/form.js:497]: LOL ok now I'm totally confused. ghe-emoji

File: src/applications/caregivers/containers/IntroductionPage.jsx [hunk 15]
```diff
@@ -23,62 +19,86 @@ const IntroductionPage = ({ route, router }) => {
...
+                  </a>{' '}
+                  or a Veterans Service Organization
+                </li>
+              </ul>
+
+              <a
+                href="https://www.va.gov/disability/get-help-filing-claim/"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
```
Comment (Mottie) [inline src/applications/caregivers/containers/IntroductionPage.jsx:99]: I don't think internal links need a domain, or `target` and `rel` attributes.

File: src/applications/caregivers/containers/IntroductionPage.jsx [hunk 16]
```diff
@@ -88,73 +108,92 @@ const IntroductionPage = ({ route, router }) => {
...
-            </div>
-            <RepresentativeInfo />
+            <p>
+              <b>Note:</b> If the Veteran isn’t enrolled in VA health care or is
+              currently on active duty with a medical discharge, they’ll need to
+              fill out an
+              <a
+                target="_blank"
+                rel="noopener noreferrer"
+                href="https://www.va.gov/health-care/how-to-apply/"
```
Comment (Mottie) [inline src/applications/caregivers/containers/IntroductionPage.jsx:123]: Another internal link

File: src/applications/caregivers/definitions/caregiverUI.js (hunk 7 above)
Comment (Mottie) [reply src/applications/caregivers/definitions/caregiverUI.js:60]: And there is an email UI definition you can build this off of: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms-system/src/js/definitions/email.js

**Review by Mottie — APPROVED** [review]

File: src/applications/caregivers/components/AdditionalInfo/index.jsx (hunk 9 above)
Comment (jenstrickland) [reply src/applications/caregivers/components/AdditionalInfo/index.jsx:58]: Thanks, @Mottie, I have not seen this. @ala-yna @sshein @jonathanjnelson can you tell me more about the decisions here? I suspect there are objectives here I am unaware of.

- modal inside of a dropdown — will introduce usability and accessibility challenges, as well as complicate QA
- link reads "Learn more about what this means" (for WCAG, the link should read Learn more about Primary Family Caregivers)

Is this a remnant of the challenge between legalese and plain language? From a 508 lens, the content in the modal **should** be the content in the additional info show/hide, and the legalese (if required here) would be at the bottom of it. If a case needs to be made with stakeholders, here are a couple of references for reasoning:

- [W3C WCAG 2.4.4 Link Purpose](https://www.w3.org/TR/WCAG20-TECHS/F63.html)

- [18f Plain Language, when to use legal and technical terms](https://content-guide.18f.gov/plain-language/#when-to-use-legal-and-technical-terms)

I hope this helps, and if I'm missing context, I'm all 👂 👂 .

**Review by maharielrosario — APPROVED** [review]

File: src/applications/caregivers/config/form.js (hunk 11 above)
Comment (LinnJS) [reply src/applications/caregivers/config/form.js]: Cleared it up with @jonathanjnelson correct it

File: src/applications/caregivers/containers/IntroductionPage.jsx (hunk 15 above)
Comment (LinnJS) [reply src/applications/caregivers/containers/IntroductionPage.jsx:99]: Fixed all occurrences

File: src/applications/caregivers/components/AdditionalInfo/index.jsx (hunk 9 above)
Comment (ala-yna) [reply src/applications/caregivers/components/AdditionalInfo/index.jsx:58]: Hi @jenstrickland! You're right, this is due to the feedback from our stakeholders regarding legalese. Our other option would be to remove the plain language all together, which seemed like a worse user experience. This is a temporary fix, designed to meet our stakeholder requirements and give the Veterans a more friendly explanation to work with. We hope to move past the modal solution once we're able to build a better relationship with the Program team. In the meantime, we will certainly be presenting the stakeholder team with this feedback during our upcoming Staging review.

Comment (jenstrickland) [reply src/applications/caregivers/components/AdditionalInfo/index.jsx:58]: Ah, I suspected as much, @ala-yna. Would it help for the stakeholders to learn Section 508 and the [Plain Writing Act of 2010](https://plainlanguage.gov/guidelines/) require this?

File: src/applications/caregivers/config/form.js (hunk 13 above)
Comment (LinnJS) [reply src/applications/caregivers/config/form.js:394]: Silly stakeholder requirements :( had to change gender to sex and add "unknown"

File: src/applications/caregivers/config/form.js (hunk 14 above)
Comment (LinnJS) [reply src/applications/caregivers/config/form.js:509]: More silly stakeholder requirements if you are commenting about the "No country" part lol

File: src/applications/caregivers/containers/IntroductionPage.jsx (hunk 16 above)
Comment (LinnJS) [reply src/applications/caregivers/containers/IntroductionPage.jsx]: So I removed all of these them the linter was complaining about not having it with `target="_blank"` so I went ahead and added it back so I could commit.

File: src/applications/caregivers/containers/IntroductionPage.jsx (hunk 15 above)
Comment (LinnJS) [reply src/applications/caregivers/containers/IntroductionPage.jsx:99]: So I removed all of these them the linter was complaining about not having it with target="_blank" so I went ahead and added it back so I could commit.

**Review by maharielrosario — APPROVED** [review]

**Review by bradl3yC — APPROVED** [review]

**Review by jhonny218 — APPROVED** [review]
LGTM

**Pull request merged**